lib/errors.js

```diff
@@ -40,10 +40,16 @@ util.inherits(error.BaseError, Error);
 error.ValidationError = function(message, errors) {
   error.BaseError.apply(this, arguments);
   this.name = 'SequelizeValidationError';
+  this.message = "Validation Error";
   this.errors = errors || [];
 
-  if (this.errors.length > 0 && this.errors[0].message) {
-    this.message = this.errors.map(function (err) {
+  // Use provided error message if available...
+  if (message) {
+    this.message = message;
+  }
+  // ... otherwise create a concatenated message out of existing errors.
+  else if (this.errors.length > 0 && this.errors[0].message) {
```

Comment:
I'd move this on line 49 so it's more apparent what's going on here `} else if (this.errors.length [...]`